docs/conf.py

```diff
@@ -28,7 +28,7 @@
 author = "Kraken Tech"
 
 # The full version, including alpha/beta/rc tags
-release = "2.2.1"
+release = "2.2.2"
```

Comment:
The addition of new feature isn't a maintenance update. Instead this is an update to `v2.3.0`.